aslprep/interfaces/cbf_computation.py

```diff
@@ -528,10 +524,29 @@ def _run_interface(self, runtime):
         m0data = masker.transform(m0_file).T
         scaled_m0data = m0scale * m0data
 
+        # Offset PLD(s) by slice times
+        if "SliceTiming" in metadata:
+            slice_times = metadata["SliceTiming"]
+            deltam_img = nb.load(deltam_file)
+            shape = deltam_img.shape[:3]
+
+            pld_brain = np.tile(plds, list(shape) + [1])
+
+            # Assumes XXI orientation?
+            pld_brain = pld_brain + slice_times[None, None, :, None]
+            pld_img = nb.Nifti1Image(pld_brain, deltam_img.affine, deltam_img.header)
```

Comment:
@mattcieslak would you mind taking a look at this? I don't how to account for the image orientation and slice encoding direction (which is an optional metadata field) when shifting the values by the slice times.